ui/components/multichain/create-account/create-account.tsx

```diff
@@ -44,19 +62,11 @@ export const CreateAccount = ({ onActionComplete }) => {
     defaultAccountName,
   );
 
-  const onCreateAccount = async (name) => {
-    const newAccountAddress = await dispatch(addNewAccount());
-    if (name) {
-      dispatch(setAccountLabel(newAccountAddress, name));
-    }
-  };
-
-  const onSubmit = async (event) => {
+  const onSubmit = async (event: KeyboardEvent<HTMLFormElement>) => {
```

Comment:
This method could be inside a `useCallback`

Reply:
ah yes, definitely :) thanks!